django/text_extractor/views.py

```diff
@@ -75,22 +77,32 @@ def submit_document(request):
                 files.insert(0, toc_file)
 
             for idx, file in enumerate(files):
-                ocr_file, txt_file, cost = create_searchable_pdf(
-                    file, merged and idx > 0
+                file.seek(0)
+                file_content = file.read()
+                file.seek(0)
```

Comment:
without file.seek(0) it would not read the next pdfs from the beginning of the file.